roles/mirror_ocp_release/tasks/facts.yml

```diff
@@ -7,12 +7,27 @@
   register: release_image
   changed_when: false
 
-- name: "Write rhcos.json from command (>= 4.8)"
+- name: Write rhcos.json from command (4.16+)
+  when: mor_version is version("4.16", ">=")
```

Comment:
The binary from the previous version can be executed using centos:stream9. 

What about using the container approach for all versions > 4.8?

Reply:
We addressed this comment offline. All good.